stable/mariadb/templates/role.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.serviceAccount.create  .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "master.fullname" . }}
+  labels:
+    app: "{{ template "mariadb.name" . }}"
+    chart: {{ template "mariadb.chart" . }}
```

Comment:
For consistency, I'd also quote this label the same way you did with "app" label. It applies to the rest of manifests.